components/pages/index.js

```diff
@@ -1,16 +1,20 @@
+import { useState } from 'react';
 import PropTypes from 'prop-types';
 import MemberList from 'components/members-list';
 import classNames from 'components/pages/home-page.module.scss';
 import NewMemberList from 'components/new-member-list';
+import SearchMembers from 'components/search-members/index';
 
 const HomePage = () => {
+  const [searchTerm, setSearchTerm] = useState('');
```

Comment:
Could you try exploring having context here, I feel it would be a better way to approach this

Reply:
ok